mailbox/api/src/test/java/org/apache/james/mailbox/MailboxManagerTest.java

```diff
@@ -223,7 +223,7 @@ public void user2ShouldBeAbleToCreateNestedFoldersWithoutTheirParents() throws B
 
```

Comment:
Your implementation of MailboxManager::hasCapability should be in this commit, not in the previous one.... It makes the review harder to do....